### ISSUE TITLE
Using Scroll Wheel in "Month" Tab

### ISSUE THREAD

**Issue by Gameniti** [marcoancona/MineTime#211]
It would be nice if you can use the mouse wheel to scroll through the months instead of using the on-screen arrows or the arrow keys. 
I don't really use the other tabs that often, but it may be also a good addition to them to add something like "infinite Scroll" where you can scroll to the next day/week/10days when you are already scrolled to the bottom of the page (so not really infinite to prevent accidentally scrolling too far).

**Comment by marcoancona** [comment]
The problem here is the conflict between scrolling "down" (if the all calendar view does not fit) and moving across days/months. Notice that sometimes scrolling down is needed also in the month view if you have many events. One could move to another day/month when continuing scrolling once the top/bottom is reached but I believe this might be confusing for some.

**Comment by marcoancona** [comment]
The problem here is the conflict between scrolling "down" (if the all calendar view does not fit) and moving across days/months. Notice that sometimes scrolling down is needed also in the month view if you have many events. One could move to another day/month when continuing scrolling once the top/bottom is reached but I believe this might be confusing for some.

**Comment by PunkIslamist** [comment]
Hey there! I would give my +1 to this feature request. I still find myself trying to scroll down the month view, even after using MineTime for a long time.
A possible solution for the conflict with scrolling down crammed days could be: only scroll inside a day when it is selected. So by default I can scroll through the months, and if I want to focus on a specific day, I select it, and then I can scroll inside that day.

**Comment by Gameniti** [comment]
> A possible solution for the conflict with scrolling down crammed days could be: only scroll inside a day when it is selected.

Yeah, that would be an option. Personally I think I would find it better to not directly select the day but to check if I hover over a day where I can scroll through. If possible you could add this as an optional setting.
